modules/data_release/templates/menu_data_release.tpl

```diff
@@ -84,6 +95,61 @@
+
+<div class="modal fade" id="managePermissionsModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
+                <h3 class="modal-title" id="myModalLabel">Manage Permissions</h3>
+            </div>
+            <form name = "uploadForm" id = "uploadForm" method = "POST" enctype="multipart/form-data" action="{$baseurl}/data_release/ajax/AddPermission.php">
+                <div class="modal-body">
+                    <div class="row">
+                        <table  class="table table-hover table-primary table-bordered dynamictable" border="0" width="100%">
+                            <thead>
+                                <tr class="info">
+                                    <th>Users</th>
+                                    <th>Permissions</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                {foreach from = $manage_permissions key=k item=elem}
+                                <tr>
+                                    <td>
+                                        {$k}
+                                        <!--<input type="hidden" name="{$k}" value="{$k}" readonly>{$k}</input>-->
+                                    </td>
+                                    <td>
+                                    {foreach from = $data_release_versions item=vv key=kk}
+                                        {foreach from = $elem key=key item=value}
+                                           {if in_array($vv, $elem)}
+                                               <input type='checkbox' name='permissions_{$k}[]' value='{$vv}' checked>{$vv}</input>
+                                           {else}
+                                               <input type='checkbox' name='permissions_{$k}[]' value='{$vv}'>{$vv}</input>
+                                           {/if}
+                                           {break} 
+                                        {/foreach}
+                                    {/foreach}
+
+                                    </td>
+                                </tr>
+                                {/foreach}
+                            </tbody>                        
+                        </table>
+                        <input type="hidden" name = "action" id = "action" value = "managepermissions">
+                    </div>
+                </div>
+                <div class="modal-footer">
+                    <button class="btn btn-primary" id = "uploadButton" role="button" aria-disabled="false">Manage Permissions</button>
```

Comment:
This is the third button with id=uploadButton
`[DOM] Found 3 elements with non-unique id #uploadButton`